.repoman.yml

```diff
@@ -54,7 +54,7 @@ issues:
         - type: query
           value: "contains(Issue.body, '.NET 7 Preview') == `true` || contains(Issue.body, '.NET 7 RC') == `true` || contains(Issue.body, '.NET 7 GA') == `true`"
       pass:
-        - labels-add: [":checkered_flag: Release .NET 7"]
+        - labels-add: [":checkered_flag: Release: .NET 7"]
```

Comment:
What about line 51 -- Release .NET 6 doesn't have a colon.

Reply:
I updated the existing .NET 6 label to add the colon and I've updated this script to match it.